back/doc/dependency_decisions.yml

```diff
@@ -172,3 +172,15 @@
     :why: 
     :versions: []
     :when: 2021-03-28 18:10:30.586364420 Z
+- - :approve
+  - brakeman
+  - :who: 
+    :why: 
+    :versions: []
+    :when: 2022-06-17 14:33:21.323236833 Z
+- - :approve
+  - bundler-audit
+  - :who: 
+    :why: 
+    :versions: []
+    :when: 2022-06-17 14:33:30.806575129 Z
```

Comment:
I checked bundler-audit's license, and it's GPLv3, which is not compatible with our licenses (see https://www.notion.so/citizenlab/Source-code-licenses-0e56954f330e41ba9bef7045b75eb1c0#c0cbc9a082594c7e93824a5f97cb9d28)

However, in this case it's a bit more subtle, since we're not 'linking' out code to theirs, but merely executing it, which makes it okay. It wouldn't be okay in case we `require`ed it somewhere and called into the library, so we need to make sure we never do this.

Best to add this to the `why:` 

```
Despite brakeman being a GPL utility, we're not linking against it, but merely executing it in its distributed form as a rake task.
```

Reply:
I added it, but replaced `brakeman` with `bundler-audit`, as that was the subject of the comment.

Reply:
Oops, yes. All those `b` tools tricking me up!